public/video-ui/src/components/utils/VideoEmbed.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export function VideoEmbed({ sources }) {
+  return (
+    <video className="video-player" controls>
```

Comment:
can we set the `preload` attribute to `metadata` to avoid downloading the video on page load